source/read/distinct.txt

```diff
@@ -0,0 +1,173 @@
+.. _java-rs-distinct:
+
+==============================
+Retrieve Distinct Field Values
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, unique, code example
+
+Overview
+--------
+
+Within a collection, different documents might contain different values for a single field.
+For example, one document in the ``restaurant`` collection has a ``borough`` value of ``"Manhattan"``, and
+another has a ``borough`` value of ``"Queens"``. With the {+driver-short+}, you can
+retrieve all the distinct values that a field contains across multiple documents
+in a collection.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see
+:ref:`<java-rs-getting-started>`.
+
+.. include:: includes/reactor-note.rst
+
+``distinct()`` Method
+---------------------
+
+To retrieve the distinct values for a specified field, call the ``distinct()``
+method and pass in the name of the field you want to find distinct values for.
+
+Retrieve Distinct Values Across a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example retrieves the distinct values of the ``borough`` field in
+the ``restaurants`` collection:
+
+.. io-code-block::
+
+   .. input:: /includes/read-ops/distinct.java
+      :start-after: start-distinct
+      :end-before: end-distinct
+      :language: java
+      :dedent:
+
+   .. output::
+      :visible: false
```

Comment:
S: I think it helps demonstrate what `distinct()` is doing when you can see the values it returns, so I'd vote we show the output by default
```suggestion
   .. output::
```